xml/System.Reflection/Assembly.xml

```diff
@@ -6323,14 +6330,16 @@ In .NET 5 and later versions, for bundled assemblies, the value returned is an e
       </ReturnValue>
       <Docs>
         <summary>Gets a value that indicates which set of security rules the common language runtime (CLR) enforces for this assembly.</summary>
-        <value>The security rule set that the CLR enforces for this assembly.</value>
+        <value>
+          .NET Framework only: The security rule set that the CLR enforces for this assembly.
+
+          .NET Core and .NET 5+: <see cref="E:System.Security.SecurityRuleSet.None" /> in all cases.
```

Comment:
Some of these are virtual methods on abstract types that can be implemented by user code. The non-runtime implementations of these methods can do whatever.